auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -358,14 +375,33 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
         }
       }
     }
-    const filingResponse = await this.createNamedBusiness(filingBody)
+    return this.createNamedBusiness(filingBody, business.businessIdentifier)
+  }
 
-    if (filingResponse?.errorMsg) {
-      this.$emit('add-unknown-error')
-      return ''
-    } else {
-      return filingResponse.data.filing.business.identifier
+  private async createBusinessRegistration (business: Business): Promise<any> {
+    const filingBody: RegistrationRequest = {
+      filing: {
+        header: {
+          name: FilingTypes.REGISTRATION,
+          accountId: this.currentOrganization.id
+        },
+        business: {
+          legalType: business.nameRequest.legalType,
+          natureOfBusiness: business.nameRequest.natureOfBusiness
+        },
+        registration: {
+          nameRequest: {
+            legalType: business.nameRequest.legalType,
+            nrNumber: business.businessIdentifier
+          }
+        }
+      }
+    }
+    if (business.nameRequest.legalType === LegalTypes.SP) {
+      filingBody.filing.registration.businessType =
+                    (business.nameRequest.entityTypeCd === 'FR' ? 'SP' : business.nameRequest.entityTypeCd)
```

Comment:
Should use an enum for Entity Type Code.

See https://github.com/bcgov/namerequest/blob/main/src/enums/entity-type.ts